src/main/java/org/opensearch/ad/ml/EntityColdStarter.java

```diff
@@ -94,9 +98,12 @@
     private final int shingleSize;
     private Instant lastThrottledColdStartTime;
     private final FeatureManager featureManager;
-    private final Cache<String, Instant> lastColdStartTime;
-    private final CheckpointDao checkpointDao;
     private int coolDownMinutes;
+    // A bloom filter checked before cold start to ensure we don't repeatedly
```

Comment:
Minor: The DoorKeeper is the bloom filter. Add comment saying keys are detector ids.

Reply:
added